README.adoc

```diff
@@ -0,0 +1,38 @@
+# Jenkins Automated Release Flow
+
+As a part of the link:https://issues.jenkins-ci.org/browse/INFRA-910[INFRA-910] EPIC,
+https://jenkins.io/projects/infrastructure/[Jenkins Infrastructure Team] is working on a new Jenkins Core Release Environment
+which would be able to release Jenkins Weekly and LTS versions in an automatic way.
+It includes packaging, staging, and releasing stages.
+
+## Jenkins Core Release Environment
+
+The release environment is hosted within the https://github.com/jenkins-infra/openvpn[Jenkins VPN].
+It includes a Jenkins instance and a number of other services.
+If you are connected to VPN, the Jenkins instance can be accessed using this URL: https://release.ci.jenkins.io/ .
+
+Helm Chart for the Jenkins Core Release Environment is available https://github.com/jenkins-infra/charts/tree/master/charts/jenkins[here].
+
+## This repository
+
+This repo contains tools needed to release the https://github.com/jenkinsci/jenkins[Jenkins core] and its components
+from the Jenkins Core Release Environment.
+The following components can be found here:
+
+* Jenkinsfiles for Jenkins packaging and Release
+* Kubernetes POD templates for Linux and Windows packaging environments
+* Kubernetes POD template for the release environment
+* Release profiles and utility scripts
+
+## Jenkins Packaging
+
+* WAR packaging is defined winthin the https://github.com/jenkinsci/jenkins[Jenkins core] repository.
```

Comment:
https://github.com/jenkinsci/packaging but my PR is not yet merged

Reply:
Not sure I understand it. WAR package itself is generated within the core at the moment. I can update the doc to reference jenkinsci/packaging as a source of codesigning scripts which run on the top of it

Reply:
Yes are totally right, jenkinsci/packaging assume that a war file is already "released"